libsolidity/CompilerContext.h

```diff
@@ -94,6 +94,8 @@ class CompilerContext
 	eth::AssemblyItem appendJumpToNew() { return m_asm.appendJump().tag(); }
 	/// Appends a JUMP to a tag already on the stack
 	CompilerContext&  appendJump(eth::AssemblyItem::JumpType _jumpType = eth::AssemblyItem::JumpType::Ordinary);
+	/// Returns an "ErrorTag"
```

Comment:
Hm, they compare as equal but are copies of different objects, so you have a point there.